python/mxnet/contrib/onnx/_import/op_translations.py

```diff
@@ -454,20 +505,49 @@ def reduce_mean(attrs, inputs, proto_obj):
     return 'mean', new_attrs, inputs
 
 def reduce_min(attrs, inputs, proto_obj):
-    """Reduce the array along a given axis by mean value"""
+    """Reduce the array along a given axis by minimum value"""
     new_attrs = translation_utils._fix_attribute_names(attrs, {'axes':'axis'})
     return 'min', new_attrs, inputs
 
 def reduce_sum(attrs, inputs, proto_obj):
-    """Reduce the array along a given axis by mean value"""
+    """Reduce the array along a given axis by sum value"""
     new_attrs = translation_utils._fix_attribute_names(attrs, {'axes':'axis'})
     return 'sum', new_attrs, inputs
 
 def reduce_prod(attrs, inputs, proto_obj):
-    """Reduce the array along a given axis by mean value"""
+    """Reduce the array along a given axis by product value"""
     new_attrs = translation_utils._fix_attribute_names(attrs, {'axes':'axis'})
     return 'prod', new_attrs, inputs
 
+def reduce_log_sum(attrs, inputs, proto_obj):
+    """Reduce the array along a given axis by log sum value"""
+    keep_dims = True if 'keepdims' not in attrs else attrs.get('keepdims')
+    sum_op = symbol.sum(inputs[0], axis=attrs.get('axes'),
```

Comment:
Why inputs[0]?

Reply:
ReduceLogSum op takes has input tensor - https://github.com/onnx/onnx/blob/master/docs/Operators.md#ReduceLogSum. And while translating this ONNX operator into MXNet, we are splitting the reduceLogSum op to ``sum`` over a given axis and then ``log`` operator.

Reply:
it has been similarly used in other operator translations, for example here - https://github.com/apache/incubator-mxnet/pull/11140/files/6f518f13413387b40a5c45d96bd2c1d2decf7804#diff-ea46c490a749b158ef8986126da6bc12R284